config/default/app_settings.json

```diff
@@ -360,9 +360,10 @@
',s]||a<=1&&['m']||a<n.m&&['mm',a]||o<=1&&['h']||o<n.h&&['hh',o]||l<=1&&['d']||l<n.d&&['dd',l];return null!=n.w&&(h=h||d<=1&&['w']||d<n.w&&['ww',d]),(h=h||u<=1&&['M']||u<n.M&&['MM',u]||c<=1&&['y']||['yy',c])[2]=t,h[3]=+e>0,h[4]=r,ta.apply(null,h)}function ra(e){return void 0===e?Ks:'function'==typeof e&&(Ks=e,!0)}function ia(e,t){return void 0!==ea[e]&&(void 0===t?ea[e]:(ea[e]=t,'s'===e&&(ea.ss=t-1),!0))}function sa(e,t){if(!this.isValid())return this.localeData().invalidDate();var n,r,i=!1,s=ea;return'object'==typeof e&&(t=e,e=!1),'boolean'==typeof e&&(i=e),'object'==typeof t&&(s=Object.assign({},ea,t),null!=t.s&&null==t.ss&&(s.ss=t.s-1)),r=na(this,!i,s,n=this.localeData()),i&&(r=n.pastFuture(+this,r)),n.postformat(r)}var aa=Math.abs;function oa(e){return(e>0)-(e<0)||+e}function la(){if(!this.isValid())return this.localeData().invalidDate();var e,t,n,r,i,s,a,o,l=aa(this._milliseconds)/1e3,u=aa(this._days),d=aa(this._months),c=this.asSeconds();return c?(e=Ne(l/60),t=Ne(e/60),l%=60,e%=60,n=Ne(d/12),d%=12,r=l?l.toFixed(3).replace(/\\.?0+$/,''):'',i=c<0?'-':'',s=oa(this._months)!==oa(c)?'-':'',a=oa(this._days)!==oa(c)?'-':'',o=oa(this._milliseconds)!==oa(c)?'-':'',i+'P'+(n?s+n+'Y':'')+(d?s+d+'M':'')+(u?a+u+'D':'')+(t||e||l?'T':'')+(t?o+t+'H':'')+(e?o+e+'M':'')+(l?o+r+'S':'')):'P0D'}var ua=ar.prototype;return ua.isValid=ir,ua.abs=Ds,ua.add=Ss,ua.subtract=Ys,ua.as=Ns,ua.asMilliseconds=Rs,ua.asSeconds=Cs,ua.asMinutes=Ws,ua.asHours=Fs,ua.asDays=Us,ua.asWeeks=Hs,ua.asMonths=As,ua.asQuarters=Es,ua.asYears=Ls,ua.valueOf=Vs,ua._bubble=Os,ua.clone=Is,ua.get=Gs,ua.milliseconds=Zs,ua.seconds=zs,ua.minutes=qs,ua.hours=$s,ua.days=Bs,ua.weeks=Xs,ua.months=Js,ua.years=Qs,ua.humanize=sa,ua.toISOString=la,ua.toString=la,ua.toJSON=la,ua.locale=ai,ua.localeData=li,ua.toIsoString=Y('toIsoString() is deprecated. Please use toISOString() instead (notice the capitals)',la),ua.lang=oi,L('X',0,0,'unix'),L('x',0,0,'valueOf'),be('x',ve),be('X',De),Ce('X',(function(e,t,n){n._d=new Date(1e3*parseFloat(e))})),Ce('x',(function(e,t,n){n._d=new Date(Pe(e))})),r.version='2.30.1',i(Bn),r.fn=ls,r.min=Kn,r.max=er,r.now=tr,r.utc=_,r.unix=us,r.months=ps,r.isDate=c,r.locale=mn,r.invalid=g,r.duration=Tr,r.isMoment=M,r.weekdays=gs,r.parseZone=ds,r.localeData=gn,r.isDuration=or,r.monthsShort=ys,r.weekdaysMin=ws,r.defineLocale=pn,r.updateLocale=yn,r.locales=vn,r.weekdaysShort=vs,r.normalizeUnits=ne,r.relativeTimeRounding=ra,r.relativeTimeThreshold=ia,r.calendarFormat=Vr,r.prototype=ls,r.HTML5_FMT={DATETIME_LOCAL:'YYYY-MM-DDTHH:mm',DATETIME_LOCAL_SECONDS:'YYYY-MM-DDTHH:mm:ss',DATETIME_LOCAL_MS:'YYYY-MM-DDTHH:mm:ss.SSS',DATE:'YYYY-MM-DD',TIME:'HH:mm',TIME_SECONDS:'HH:mm:ss',TIME_MS:'HH:mm:ss.SSS',WEEK:'GGGG-[W]WW',MONTH:'YYYY-MM'},r}()}},t={};function n(r){var i=t[r];if(void 0!==i)return i.exports;var s=t[r]={id:r,loaded:!1,exports:{}};return e[r].call(s.exports,s,s.exports,n),s.loaded=!0,s.exports}return n.nmd=e=>(e.paths=[],e.children||(e.children=[]),e),n(344)})())); return ContactSummary;",
```

Comment:
I think you inadvertently merged an updated version of the config. Can you please revert this change? The unit tests for `shared-libs/rules-engine` are now failing and I believe this is the case.